cmd/snap/cmd_snap_op.go

```diff
@@ -46,7 +46,7 @@ var (
 	pollTime    = 100 * time.Millisecond
 )
 
-func wait(client *client.Client, id string) (*client.Change, error) {
+func wait(cli *client.Client, id string) (*client.Change, error) {
```

Comment:
`id` is probably change ID but this is not apparent. Perhaps it could be a dedicated type?